pom.xml

```diff
@@ -44,6 +44,9 @@
 	<profiles>
 		<profile>
 			<id>full</id>
+			<activation>
+				<activeByDefault>true</activeByDefault>
```

Comment:
All the magical is here... As we call the sonar step in a separated mvn, it shall have some modules by default..